src/components/Ditto.tsx

```diff
@@ -0,0 +1,39 @@
+import { useContext } from "react";
+import { DittoText, DittoTextProps } from "./DittoText";
+import { DittoFrameOrBlock, DittoFrameOrBlockProps } from "./DittoFrameOrBlock";
+import { DittoContext } from "../lib/context";
+import {fragmentError} from "../lib/utils";
+
+interface DittoProps {
+  projectId?: string;
+  textId?: string;
+  frameId?: string;
+  blockId?: string;
+  children?: React.ReactNode;
+}
```

Comment:
Ahhh nice catch here! 

A passed `filters` prop actually **would** automatically get passed down to `DittoFrameOrBlock`, so this would work fine for any vanilla js users, but any TypeScript users using the SDK would get an error when trying to pass `filters` as a prop!

Reply:
Fixed! afe7430